plugins/spring-boot/info/info.ts

```diff
@@ -0,0 +1,6 @@
+namespace SpringBoot {
+
+  export class Info {
```

Comment:
lets do something here. that's why you needed `_.toPairs()` in controller. Instead, convert the raw data into this Info object in service and directly use the data structure in html.

Reply:
did it in the new commit

Reply:
where is it?